Ethereal Artefacts/Ethereal Artefacts/Views/Components/SearchField.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Search.swift
+//  Ethereal Artefacts
+//
+//  Created by Albert Khurshudyan on 8.05.23.
+//
+
+import SwiftUI
+
+struct SearchField: View {
```

Comment:
There is a Search Bar that is build in in SwiftUI you can have a look ".searchable"

Reply:
Wherever I add .searchable in Homepage, it appears at the top and I can't change its position

Reply:
It is on the top of the page in the design also. It will always be beneath the Navigation Bar if there is one. But no worries, this is part of a next homework anyway - you can try to add `.searchable` then.

Reply:
Okay, thanks!